Gemfile.lock

```diff
@@ -750,6 +751,9 @@ GEM
       hashery (~> 2.0)
       ruby-rc4
       ttfunk
+    pry (0.14.1)
```

Comment:
does pry need to be updated like this in the gemfile?

Reply:
Added pry gem to Gem File for debugging, Above changes result of pry bundling.

Reply:
I'm only going on what I've heard from other rails devs, but I know there have been issues caused with adding Pry in the past -- can we confirm this has no adverse effect on production?

cc @bill-transue

Reply:
Yea, good catch @markgoho. Let's get `pry` out of the `Gemfile` please.

Reply:
removed pry gem from gem file